app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/payment_methods/MainPaymentsView.kt

```diff
@@ -51,13 +51,13 @@ import com.aptoide.android.aptoidegames.theme.Palette
 
 const val paymentsRoute = "payments"
 
-fun NavGraphBuilder.paymentsScreen(
-  navigate: (String) -> Unit,
+fun paymentsScreen(
   onFinish: (Boolean) -> Unit,
   purchaseRequest: PurchaseRequest?,
-) = staticComposable(
-  paymentsRoute,
-) {
+) = ScreenData.withAnalytics(
+  route = paymentsRoute,
+  screenAnalyticsName = "CreditCard"
```

Comment:
should this be the name here ? Not just Payments ? i understand the flow would lead only to credit card from here, but i just dont know if it isnt too specific.

Reply:
Fixed